common/thorhelper/thorsoapcall.cpp

```diff
@@ -2504,13 +2524,15 @@ class CWSCAsyncFor : implements IWSCAsyncFor, public CInterface, public CAsyncFo
                 {
                     if (master->timeLimitExceeded)
                     {
+                        clientSpan->recordError("time_limit_exceeded");
```

Comment:
I think it would be a good idea to always report an error number.
The error message should be human readable - that seems to be the implication of https://opentelemetry.io/docs/specs/semconv/exceptions/exceptions-spans/.